.github/ISSUE_TEMPLATE/story-issue-template-.md

```diff
@@ -1,38 +1,65 @@
 ---
 name: Story issue template
 about: Starting point for new stories
-title: '[distinguishable summary]'
+title: 'clear, concise summary'
 labels: ''
 assignees: ''
 ---
-# User Story
 
-In order to [goal], [stakeholder] wants [change].
+# At a glance
+
+[comment]: # "Begin with a short summary of this ticket, so its intent can be understood at a glance."
+
 
 # Acceptance Criteria
 
-[ACs should be clearly demoable/verifiable whenever possible. Try specifying them using [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications).]
+We use [behavior-driven development](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications) wherever possible.
+
+[comment]: # "ACs should be clearly demoable/verifiable whenever possible. Try specifying them using behavior driven development."
+[comment]: # "**Given** **when** **then**"
+[comment]: # "Given: the initial context at the beginning of the scenario"
+[comment]: # "when: the event that triggers the scenario"
+[comment]: # "then: the expected outcome(s)"
+[comment]: # "Repeat scenarios as needed, or repeat behaviors and lists within a scenario as needed."
+
+### Scenario: 
+
+**Given**   
+**when**  
+...
+
+```[tasklist]
+### then... 
+- [ ] 
```

Comment:
```suggestion
- [ ] [a verifiable outcome]
```

Reply:
Admittedly optimizing "for me." I don't like having text I have to remove in order to start writing. However, I did add a comment that reminds us that these items need to be verifiable.